.github/workflows/R-CMD-check.yaml

```diff
@@ -18,29 +18,48 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          # - {os: macOS-latest,   r: 'release'}
-          - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
-
+          - {os: macOS-latest,   r: 'release'}
+          #- {os: windows-latest, r: 'release'}
```

Comment:
@adokter why are `windows-latest` and `ubuntu-latest:release` disabled (and `ubuntu-latest:oldrel-1` removed)?

Is there a plan to re-enable these? If so, what is the blocker? CRAN release of vol2birdR?

Reply:
* I think the ubuntu releases can be restored, I just wasn't sure if they were adding anything
* windows-latest blocker is indeed the CRAN release of vol2birdR, we need to have the binary available

Reply:
I reopened this branch so you can make changes as you see fit

Reply:
added those other ubuntu releases